index.bs

```diff
@@ -334,6 +374,22 @@ partial interface NeuralNetworkContext {
 };
 </script>
 
+### transpose ### {#api-neuralnetworkcontext-transpose}
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand transpose(Operand input, optional sequence<long> permutation);
+};
+</script>
+<div algorithm=transpose>
+    **Arguments:**
+        - *input*: an {{Operand}}. The input N-D tensor.
+        - *permutation*: an optional sequence of {{long}} values. The values used to permute the output shape. When it's not specified, it's set to `[N-1...0]`, where `N` is the rank of the input tensor. This causes the output to become a transposed tensor of the input. When specified, the number of values in the sequence must be the same as the rank of the input tensor. 
```

Comment:
How about adding ", which must be a permutation of the sequence [1,2,...,N]." after "an optional sequence of long values"?

Reply:
Thanks @gramalingam. I understand your feedback and have updated it differently in my new commit. Please have a look if that's ok.